src/main/java/org/geysermc/discordbot/commands/GithubCommand.java

```diff
@@ -0,0 +1,92 @@
+package org.geysermc.discordbot.commands;
+
+import com.jagrosh.jdautilities.command.SlashCommand;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import org.geysermc.discordbot.GeyserBot;
+import org.geysermc.discordbot.util.BotColors;
+import org.geysermc.discordbot.util.MessageHelper;
+import org.kohsuke.github.*;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class GithubCommand extends SlashCommand {
+
+
+    private static final Pattern REPO_PATTERN = Pattern.compile("(^| )([\\w.\\-]+/)?([\\w.\\-]+)( |$)", Pattern.CASE_INSENSITIVE);
+
+    public GithubCommand() {
+        this.name = "github";
+        this.arguments = "<repo>";
+        this.help = "Get info about a given GitHub repo.";
+        this.guildOnly = false;
+        this.options = Collections.singletonList(
+                new OptionData(OptionType.STRING, "repo", "The repository to lookup, defaults to GeyserMC/Geyser")
+        );
+    }
+
+    @Override
+    protected void execute(SlashCommandEvent event) {
+        // Repo
+        String repo =  Objects.requireNonNull(event.getOption("repo")).getAsString();
+        try {
+            event.replyEmbeds(handle(repo)).queue();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private MessageEmbed handle(String repoString) throws IOException {
```

Comment:
Where is IOException thrown that we don't already catch below?

Reply:
The getLicense and getCreatedAT. Unless I try { catch it adding it there was a better option I thought